### PR TITLE
Fixes hide/show toggle issue

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@jzoldak @benpatterson @Muddasser 
https://screencast.com/t/mcLJbRh8UCMV
Please review. Fixed the test's flakiness and restructured as per discussion.

File: regression/pages/studio/pages_page_studio.py [hunk 1]
```diff
@@ -144,23 +144,21 @@ def click_and_verify_see_an_example(self):
...
-            toggle_checkbox
-        ).click().perform()
-        sync_on_notification(self)
-        # Complicated query, so executing using jQuery.
-        return self.browser.execute_script(
-            "return $('{}:eq({})').parents().eq(3).find("
-            "'.course-nav-item-header .title').text()".format(
-                toggle_checkbox_css, index
-            )
+        toggle_checkbox_css = '.is-movable[data-tab-id="wiki"] ' \
```
Comment (jzoldak) [inline regression/pages/studio/pages_page_studio.py]: Yeah, that should fix it. :)

File: regression/pages/studio/pages_page_studio.py [hunk 2]
```diff
@@ -144,23 +144,21 @@ def click_and_verify_see_an_example(self):
             'img[alt="Preview of Pages in your course"]', 'Pop up visibility'
         )
 
-    def click_hide_show_toggle(self, index=0):
+    def click_hide_show_toggle(self):
```
Comment (jzoldak) [inline regression/pages/studio/pages_page_studio.py]: nit: should probably rename this method. Something like toggle_wiki_page_display.
Oh, and update the docstring also.

File: regression/pages/studio/pages_page_studio.py [hunk 3]
```diff
@@ -181,7 +179,7 @@ def get_all_pages_count(self):
             ).results
         )
 
-    def is_page_configured_to_show(self, index=0):
+    def is_page_configured_to_show(self):
```
Comment (jzoldak) [inline regression/pages/studio/pages_page_studio.py]: nit: same here, update the method name and docstring.

**Comment by raeeschachar** [comment]
@jzoldak 
Done. Merge-able?

**Pull request merged**